blacklight-cornell/app/views/single_search/index.html.erb

```diff
@@ -9,8 +9,7 @@
               <%= render 'single_search/search_form' %>
             </div>
             <div class="advanced-search-link col-md-4">
-              <%= link_to 'Catalog Advanced Search', '/advanced' %>
-                    <%#= params[:q].inspect %>
+              <%= link_to 'Catalog Advanced Search', "/advanced?q=#{params[:q]}" %>
```

Comment:
Advanced search expects`q_row` instead of `q` as its query param (vs simple search that uses `q`). Some of the strangeness with the "Start over" button and constraints display in the Advanced Search form may be because of the q param getting populated both from bento and from simple search.

If you look at other places in bento search results, you'll actually see populated links to advanced search using q_row, e.g.: https://catalog-int.library.cornell.edu/search?q=testing which has a link to "Advanced Search" at the bottom of the Videos bento box: https://catalog-int.library.cornell.edu/edit?f%5Bformat%5D%5B%5D=Video&op_row%5B%5D=AND&q_row%5B%5D=testing&search_field_row%5B%5D=all_fields

Reply:
I ended up pushing a change to remove that param, It worked as is since I had other logic mapping q and q_row, but It was a reminder I wanted it to be more dynamic and work with the js logic I had built up.